### PR TITLE
PCX-3761: Delete browserstack app live apps when pr is closed

### PR DESCRIPTION
**Pull request by Fbada006**
## Jira ticket
[PCX-3761](https://optile.atlassian.net/browse/PCX-3761)

## Overview/What
Once a PR has been closed or moved to draft, we delete the binary on app live

## Cause/Why
To avoid having too many binaries on app live

## Solution
Run a GitHub workflow to clean up the binaries once the PR is closed

## Type of change
- Code cleanup

## Tests
N/A

### CODE REVIEW

File: .github/workflows/app_live_delete_examplecheckout.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Cleanup Browserstack
+
+on:
+  pull_request:
+    types:
+      - converted_to_draft
+      - closed
+
+jobs:
+  upload-app-live:
```
Comment (siebeprojects) [inline .github/workflows/app_live_delete_examplecheckout.yml:10]: This should be delete-app-live

File: .github/workflows/app_live_delete_examplecheckout.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        with:
+          submodules: true
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Delete example-checkout from App Live
+        env:
+          MOBILE_MERCHANT_CODE: ${{ secrets.MOBILE_MERCHANT_CODE }}
```
Comment (siebeprojects) [inline .github/workflows/app_live_delete_examplecheckout.yml:28]: Only the following secrets are needed
BROWSERSTACK_USER
BROWSERSTACK_KEY

File: .github/workflows/app_live_delete_examplecheckout.yml (hunk 1 above)
Comment (Fbada006) [reply .github/workflows/app_live_delete_examplecheckout.yml:10]: Alright

File: .github/workflows/app_live_delete_examplecheckout.yml (hunk 2 above)
Comment (Fbada006) [reply .github/workflows/app_live_delete_examplecheckout.yml:28]: Changing and retesting this

**Review by caiomello — APPROVED** [review]

**Review by siebeprojects — APPROVED** [review]

**Pull request merged**